react/src/lib/components/DeckGLMap/DeckGLMap.tsx

```diff
@@ -96,6 +97,7 @@ const DeckGLMap: React.FC<DeckGLMapProps> = ({
     getCameraPosition,
     triggerHome,
     triggerResetMultipleWells,
+    isLassoSelectionAvailable,
```

Comment:
I think we can toggle lasso selection by changing the visibility state of the layer, then we don't need this prop.